andela/travel_tool_front
# #160989966 Display requester avatar on approve-request modal

#### What does this PR do?
This PR displays the requester avatar on the approve-request modal

#### Description of Task to be completed?
- Display requester avatar on the approve-request modal

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Check out this branch with `git checkout bg-request-details-avatar-160989966`
- Run the server `make start`
- Login and navigate to the approval page
- Click on any request details on the page
- The picture of the request should be displayed.

#### Any background context you want to provide?
- You need to clone the [backend](https://github.com/andela/travel_tool_back.git) to test this functionality.

#### What are the relevant pivotal tracker stories?
[#160989966](https://www.pivotaltracker.com/story/show/160989966)

#### Screenshots (if appropriate)
<img width="1408" alt="screen shot 2018-10-04 at 3 50 23 pm" src="https://user-images.githubusercontent.com/27569442/46482720-0d0e8780-c7ee-11e8-82aa-c7966197d01e.png">


#### Questions:
N/A
